src/Compilers/CSharp/Test/Semantic/SourceGeneration/SyntaxAwareGeneratorTests.cs

```diff
@@ -1404,15 +1404,15 @@ class C
 
             var testGenerator = new PipelineCallbackGenerator(context =>
             {
-                var source = context.Sources.Syntax.Transform(c => c is FieldDeclarationSyntax fds, c =>
+                var source = context.SyntaxProvider.CreateSyntaxProvider(c => c is FieldDeclarationSyntax fds, c =>
                 {
                     syntaxCalledFor.Add(((FieldDeclarationSyntax)c.Node).Declaration.Variables[0].Identifier.ValueText);
                     return ((FieldDeclarationSyntax)c.Node).Declaration.Variables[0].Identifier.ValueText;
                 });
 
-                var source2 = source.Join(context.Sources.AdditionalTexts)
-                                    .Join(context.Sources.AnalyzerConfigOptions)
-                                    .Join(context.Sources.ParseOptions);
+                var source2 = source.Associate(context.AdditionalTextsProvider.AsSingleValue())
```

Comment:
Join => Associate. Can only associate with a single value, but there is now .AsSingleValue() that enables that